tests/cypress/integration/api/getUser.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { apolloClient } from '../../support/apollo'
+import { DocumentNode } from 'graphql'
+
+describe('Validate ability get current User', () => {
+    let GQL_APIUSER: DocumentNode
+
+    before('load graphql file', function () {
+        GQL_APIUSER = require(`graphql-tag/loader!../../fixtures/getApiUser.graphql`)
+    })
+
+    it('Get Current user for Authenticated user (jay)', async function () {
+        const response = await apolloClient({ username: 'jay', password: 'password' }).query({
```

Comment:
Wondering are we testing if GraphQL apollo client is working as intended? 
Do we want to do more white-box testing? (Apollo client is the middle-ware but ideally the API should be light and should work even without the client no?) As we have the OSGI on the bottom

Reply:
the test here tests our graphql end point, the client executes the call by providing the query with the `GQL_APIUSER`

Reply:
I assume this repo will be only sitemap specific (Is it possible to create one for sitemap as a sample?)

Reply:
I was going to do that when i had a use-case, as of now I don't have an up to date expected behavior or functionality .
That's why i added these examples, so they can be adapted by anyone, not just me.
I expect all of these examples to be gone in the next few days, once the team actually starts filling out the tests.